app/socket-apis/__tests__/get-user-post-ranks-and-top-ranked-whys.js

```diff
@@ -0,0 +1,128 @@
+// https://github.com/EnCiv/civil-pursuit/issues/208
+
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import { Mongo } from '@enciv/mongo-collections';
+import { ObjectId } from 'mongodb';
+import Rankings from '../../models/rankings';
+import Points from '../../models/points';
+import getUserPostRanksAndTopRankedWhys from '../get-user-post-ranks-and-top-ranked-whys';
+
+let memoryServer;
+let db;
+
+const synuser = { synuser: { id: new ObjectId().toString() } };
+
+beforeAll(async () => {
+  memoryServer = await MongoMemoryServer.create();
+  const uri = memoryServer.getUri();
+  await Mongo.connect(uri);
+  db = Mongo.db;
+  Rankings.setCollectionProps();
```

Comment:
Calling .setCollectionProps is not needed for these models.  It's done when the model is loaded.